package.json

```diff
@@ -14,7 +14,7 @@
     "index.cjs.js"
   ],
   "scripts": {
-    "build": "rollup index.js -f commonjs -o index.cjs.js",
+    "build": "rm -rf index.d.ts && tsc && rollup index.js -f commonjs -o index.cjs.js",
```

Comment:
Sorry, I don't know typescript. Is this typically how this is done or is the `.d.ts` usually checked in?